daisy_workflows/image_import/enterprise_linux/translate.py

```diff
@@ -161,8 +162,29 @@ def DistroSpecific(g: guestfs.GuestFS):
 
   logging.info('Updating initramfs')
   for kver in g.ls('/lib/modules'):
+    # Although each directory in /lib/modules typically corresponds to a
+    # kernel version  [1], that may not always be true.
+    # kernel-abi-whitelists, for example, creates extra directories in
+    # /lib/modules.
+    #
+    # Skip building initramfs if the directory doesn't look like a
+    # kernel version. Emulates the version matching from depmod [2].
+    #
+    # 1. https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/lib.html
+    # 2. https://kernel.googlesource.com/pub/scm/linux/kernel/git/mmarek/kmod
+    # /+/tip/tools/depmod.c#2537
+    if not re.match(r'^\d+.\d+', kver):
+      logging.debug('Skipping {}'.format(kver))
```

Comment:
minor: Add what is being skipped to the message

Reply:
done